xml/System.Net.Http/HttpClient.xml

```diff
@@ -566,12 +566,18 @@ The `DefaultRequestVersion` property can be changed as long as the <xref:System.
   
 ## Remarks  
  This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after the whole response (including content) is read.  
-  
+
+> [!NOTE]
+> In case of timeout different types of exceptions are thrown on different .NET platform versions.
```

Comment:
```suggestion
> In case of timeout, different exceptions are thrown on different .NET implementations.
```
@tdykstra Is "implementations" the recommended word to differentiate between Framework/Core/5?